core/import-csv.c

```diff
@@ -178,7 +178,7 @@ static int parse_dan_format(const char *filename, struct xml_params *params, str
 		memset(tmpbuf, 0, sizeof(tmpbuf));
 		iter = strchr(iter, '|');
 
-		if (iter && iter[1]) {
```

Comment:
so this one makes sense. As it checks if `iter` is a point, and if the second character is not `'\0'`
Removing this seems potentially wrong, because now we could call `xml_params_add()` with an empty string which I don't think is what we would want here...

Reply:
Doesn't really make sense to me.

The code looks like this:
- Find '|'.
- If found, find another '|' after that
- If that is also found, do something with the string between both '|'s.

Why would it matter if the first '|' is the last character in the string?

And farther down, it just searches for the end of the line anyway.